src/templates/BuildingOwner.vue

```diff
@@ -93,14 +108,45 @@ export default class BiggestBuildings extends Vue {
   calculateOwnedBuildingStats(): void {
     let totalGHGEmissions = 0;
     let totalGHGIntensity = 0;
+    let totalSquareFootage = 0;
+    let totalYearBuilt = 0;
+    let buildingsWithYear = 0;
+    const gradeCounts: Record<string, number> = {
+      A: 0,
+      B: 0,
+      C: 0,
+      D: 0,
+      F: 0,
+    };
 
     this.buildingsFiltered.forEach((buildingEdge: IBuildingEdge) => {
       const building: IBuilding = buildingEdge.node;
 
       totalGHGIntensity += building.GHGIntensity;
       totalGHGEmissions += building.TotalGHGEmissions;
+      totalSquareFootage += building.GrossFloorArea || 0;
+
+      // Calculate average building age
```

Comment:
@JonHunt1995 - this is the code that generates our new stats for the building owner page. You could generalize this for wards!